crates/core/tedge_mapper/src/main.rs

```diff
@@ -111,11 +124,88 @@ async fn main() -> anyhow::Result<()> {
         )?);
     }
 
+    let mut mqtt_actor = get_mqtt_actor(component.session_name(), &config).await?;
+    let mut runtime = Runtime::try_new(runtime_events_logger).await?;
+    let mut signal_actor = SignalActor::builder();
     if mapper_opt.init {
         component.init(&mapper_opt.config_dir).await
     } else if mapper_opt.clear {
         component.clear_session().await
+    } else if component.session_name().eq("tedge-mapper-az") {
```

Comment:
I'd have moved this `if component.session_name().eq("tedge-mapper-az")` check inside the `else` block as I find long if-else arms checking various different things in each arm difficult to follow. It'd be much cleaner if the top-level if-else branching is just checking the `mapper_opt` variants and an inner if-else branch can check the mapper type.